config/options.json

```diff
@@ -315,22 +340,24 @@
       "custom": "Custom (Specify...)"
     }
   },
-  "ohko": {
-    "name": "One-hit KO",
+  
+  "bossElementRando": {
+    "name": "Randomize Boss Elements",
```

Comment:
If it's spelled/capitalized like that in the randomizer, let's leave it like this then I guess.

Otherwise https://github.com/sbzappa/marabot/pull/63#discussion_r805171310 is also a typo.

Reply:
I have changed that typo. As this is a misspelling in the randomizer then, I'll forward it to the dev as well.
This should mean that everything is resolved now.